app/views/categories/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= render "shared/nav_bar"%>
+<%= render "shared/messages"%>
+<div class="d-flex justify-content-between m-1 p-1">
+  <p class= "fs-5"><%=@category.name%> category expenses:</p>
+  <p class= "fs-5"><%= number_to_currency(@transacctions.sum(:amount))%></p>
+</div>
+
+<% @transacctions.each do |item|%>
+    <div class="d-flex justify-content-between m-1 p-2 border border-secondary">
+      <div>
+        <p class= "text-dark fs-5 m-0"><%= item.name%></p>
+        <p class= "text-dark fs-6"><%= item.created_at.strftime("%d %b %y")%></p>
+      </div>
+      <p class= "text-center text-dark fs-5"><%= number_to_currency(item.amount)%></p>
+    </div>
+<%end%>
+
+<%= link_to 'Add transaction', new_category_transacction_path(@category), class:'my-3 btn btn-primary w-100 rounded-0 fs-5' %>
```

Comment:
- [ ] Kindly improve the design of the show page based on the design. You can follow the suggestions listed in the related comment above.

  Your Implementation | Behance Design
  :---:|:---:
  ![bhw](https://user-images.githubusercontent.com/36542986/220814850-3b3fdac6-4633-4367-a0ce-043cfcdf47ad.jpg) | ![riew](https://user-images.githubusercontent.com/36542986/220814853-ca90ae92-1801-4be3-8b88-a4ac26ba2962.jpg)